ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -139,21 +225,23 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
                         }
                     }
 
-                    log.info("Streaming ingestion failed, trying again", e);
+                    log.info(String.format("Streaming ingestion failed, trying again after sleep of %s +~ %s seconds", retry.getCurrentSleepSecs(),
```

Comment:
I think this would be best encapsulated within `ExponentialRetry`. Here we have an external class tasked with printing the state of that class.

I'm not sure why anyone else even needs access to `currentSleepSecs` and `maxJitterSecs`. I don't think I would provide a getter for them.